specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/redis.json

```diff
@@ -1796,6 +1796,11 @@
           "type": "integer",
           "format": "int32",
           "description": "If clustering is enabled, the Shard ID of Redis Instance"
+        },
+        "isMaster": {
```

Comment:
@Priya91 I didn't think we were going to add this in the existing api-version. As the old guidelines I am familiar with are to do a new api version if we want to add fields to responses.

Reply:
Ah, perhaps its because still marked as a preview api version for AZ - after all this time.

Reply:
I think it might also be about time we promote this api-version to stable.

Reply:
Yeah, since it's still in preview, didnt need api version update.